src/components/dialogs/parameters/sensitivity-analysis-parameters.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useCallback } from 'react';
+import { Grid } from '@mui/material';
+import { CloseButton, DropDown, LabelledButton, useStyles } from './parameters';
+import { LineSeparator } from '../dialogUtils';
+
+export const SensitivityAnalysisParameters = ({
+    hideParameters,
+    parametersBackend,
+}) => {
+    const classes = useStyles();
+
+    const [providers, provider, updateProvider, resetProvider] =
+        parametersBackend;
+
+    const updateProviderCallback = useCallback(
```

Comment:
we can extract the common functionality of updateProviderCallback into a separate function, and use it in the DropDown component.
like this : 
```
const handleUpdateProvider = (evt) => updateProvider(evt.target.value);

const updateProviderCallback = useCallback(handleUpdateProvider, [updateProvider]);
```

Reply:
Ok